src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -48,7 +48,12 @@ public static Command parse(String fullCommand) throws DukeException {
                 throw new DukeException(Messages.WARNING_NO_TASK);
             }
         case ListCommand.COMMAND_WORD:
-            return new ListCommand();
+            if (words.length == 1) {
+                return new ListCommand();
+            } else {
+                int priority = Integer.parseInt(words[1]);
+                return new ListCommand(priority);
```

Comment:
I think this part may throw an exception when the user doesn't enter an integer input.

Reply:
Otherwise I think it looks good to merge 👍

Reply:
Oh I didn't see this comment just now before I merge PR. I will change it later.